platform/tezos/stake.go

```diff
@@ -97,3 +101,22 @@ func normalizeValidator(v Validator) (validator blockatlas.Validator) {
 		Details: getDetails(),
 	}
 }
+
+func getUnknownValidator(address string) blockatlas.StakeValidator {
+	return blockatlas.StakeValidator{
+		ID:     address,
+		Status: false,
+		Info: blockatlas.StakeValidatorInfo{
+			Name:        "Decommissioned",
```

Comment:
constant?

Reply:
As a Cosmos, this path should be removed in a future.